source/Phrase.cpp

```diff
@@ -13,11 +13,31 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 #include "Phrase.h"
 
 #include "DataNode.h"
+#include "Format.h"
 #include "GameData.h"
 #include "Random.h"
 
 using namespace std;
 
+namespace {
+	// Replace oldStr with newStr in target.
+	string Replace(const string &target, const string &oldStr, const string &newStr)
+	{
+		const size_t oldSize = oldStr.size();
+		const size_t newSize = newStr.size();
+		string result(target);
```

Comment:
Use { and } for direct initialization, or use copy-initialization since we know a copy has to occur here anyway.

Reply:
I'll fix it:
```
string result = target;
```